src/index.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable import/no-mutable-exports */
+import './style.css';
+
+const submit = document.querySelector('.add-btn');
+const tasksDiv = document.querySelector('.tasks');
+export const taskInput = document.querySelector('.add-task input');
+
+let editId;
+let isEditTask = false;
+let tasks = JSON.parse(localStorage.getItem('task-list'));
```

Comment:
I would recommend reducing the number of global variables in a given project. And if need be of using more than one global variable then I would recommend closures. What do you think?